packages/core/theme/src/controllers/xm-theme-controller.service.ts

```
@@ -83,19 +86,30 @@ export class XmThemeController implements OnDestroy {
 
-        if (theme.themeStrategy === 'TENANT_ONLY') {
-            this.styleManager.set('theme', `assets/css/${file}.css`);
-        } else {
-            this.styleManager.set('theme', `/assets/themes/${file}.css`);
+        let themeHref = '';
+        switch (themeStrategy) {
+            case 'TENANT_ONLY':
+                themeHref = `assets/css/${file}.css`;
+                break;
+            case 'CUSTOM':
+                const {themePath} = this.publicConfig() || {};
```

Biome (2.5.3)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)